agents/plugins/mk_oracle

```diff
@@ -1749,7 +1760,7 @@ sql_instance () {
                         where p.name = 'enable_pluggable_database'
                       union all
                       select
-                             i.instance_name, i.version, i.status, i.logins, i.archiver
+                             i.instance_name, i.${version_column} version, i.status, i.logins, i.archiver
```

Comment:
i.instance_name, i.${version_column}, i.status, i.logins, i.archiver

Reply:
We could remove the version, due to the 'union all'. We are not at the 1st part of the union all who defines the column names in the SQL.
I am for keeping it. That makes the understanding of the SQL easier.

Reply:
@lasoe So would it be OK for you to keep `i.${version_column} version`?

Reply:
@jherbel I agree with Thorsten, keeping it makes it more understandable.